src/URLShortener.Shared/Services/Interfaces/IVisitService.cs

```diff
@@ -0,0 +1,13 @@
+using URLShortener.Domain.Entities;
+using URLShortener.Shared.Models.Visit;
+
+namespace URLShortener.Shared.Services.Interfaces;
+
+public interface IVisitService
+{
+        public Task<List<Visit>> GelAllByLinkId(GelAllByLinkIdModel model);
+        public Task<ICollection<Visit>> GetAllByUserId(GetAllByUserIdModel model);
+        
+        public Task<Visit> CreateVisit(CreateVisitModel model);
+}
```

Comment:
Forgive me for my mistake but I think we shoud create another method to update visit data for (City and Country). 
(e.g UpdateVisitGeoData)